src/transforms/remap.rs

```diff
@@ -122,9 +136,10 @@ impl FunctionTransform for Remap {
 
         let mut target: VrlTarget = event.into();
 
-        let mut runtime = Runtime::default();
-
-        let result = runtime.resolve(&mut target, &self.program, &self.timezone);
+        self.runtime.clear();
+        let result = self
+            .runtime
+            .resolve(&mut target, &self.program, &self.timezone);
```

Comment:
It's probably a good idea to add a test to validate that this keeps working as expected (e.g. given two events, the runtime only keeps its state around per event)

Reply:
I added a simplistic test that I think at least confirms that variable assignments don't cross the invocation boundary?